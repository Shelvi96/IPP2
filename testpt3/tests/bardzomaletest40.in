NEW ID

0737:7>6;79133
61;;502826>5;
527096>56985
981:6215;>;886119
637;02190>9:0;66950
20>2050169:
09843964>3282024
3;43>631:8
764;80>;217:5
6;61>74
;;2963723>17598
02:00716>;7673166
80>37::
@39541;71;67:56905514;80473;5
@;398051403372100306852722;9927
@124969;:26698;98226:;8
@1293402;14332461607306509;7;:
@942;;6;15450;745039;616327
@1650812700:15
@:96923:519367;82511229388
@9682
@64:862:377:1712173299356;
@7991275
@7;65651978:1958476;
@434;0
@0656;099683;
@:;3301;405881;003:5:3;130:1810
@3;96026823425729;421
@523608629890944;2555
@7291486:95
@3023909607:933058929496:168
