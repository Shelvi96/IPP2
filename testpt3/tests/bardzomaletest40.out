763498980296470
91969387023868618
512
8389335230324708
2000011000000
0
101312100000
0
573647556900
0
118261
0
0
281474976710656
3545818
532179
0
34886069934397
